### ISSUE TITLE
build failure with latest nightly

### ISSUE THREAD

**Issue by matthiaskrgr** [rust-lang-nursery/rust-semverver#68]
`rustc 1.30.0-nightly (ae7fe84e8 2018-09-26)`
````
   Compiling git2-curl v0.8.1
   Compiling cargo v0.29.0
   Compiling semverver v0.1.17
error[E0061]: this function takes 4 parameters but 3 parameters were supplied
   --> /home/matthias/.cargo/registry/src/github.com-1ecc6299db9ec823/semverver-0.1.17/src/semcheck/typeck.rs:217:24
    |
217 |             self.infcx.resolve_regions_and_report_errors(target_def_id,
    |                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected 4 parameters

error: aborting due to previous error

For more information about this error, try `rustc --explain E0061`.
error: failed to compile `semverver v0.1.17`, intermediate artifacts can be found at `/tmp/cargo-installiss47U`

Caused by:
  Could not compile `semverver`.
````

**Comment by ibabushkin** [comment]
This looks simple to fix, I'll do that today and publish a new version on crates.io. Thanks for reporting!

**Comment by ibabushkin** [comment]
Fixed in version 0.1.18 on crates.io.
